lib/routes/gs-developer/blog.ts

```diff
@@ -0,0 +1,64 @@
+import { Route, Data } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/blog',
+    categories: ['blog'],
+    example: '/gs-developer/blog',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['developer.gs.com/blog/posts'],
+            target: '/blog',
+        },
+    ],
+    name: 'blog',
+    zh: {
+        name: '博客',
+    },
+    maintainers: ['chesha1'],
+    handler: handlerRoute,
+};
+
+async function handlerRoute(): Promise<Data> {
+    const response = await ofetch('https://developer.gs.com/blog/posts', {
+        headers: {
+            'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36',
+        },
+        body: null,
```

Comment:
RSSHub do use a randomised UA
https://github.com/DIYgod/RSSHub/blob/69cbeae1826b1107523903beebe8cbb9335917ad/lib/config.ts#L380
Are there any specific reasons to use this fixed UA string instead of a randomised one?

Reply:
No other reasons, just because I do not know rsshub will have randomised UAs.

And this does not work in my local environment.  Does this randomised UA only work in rsshub server side?